src/sentry_sdk.h

```diff
@@ -0,0 +1,57 @@
+#ifndef SENTRY_SINGLETON_H
+#define SENTRY_SINGLETON_H
+
+#include "runtime_config.h"
+#include "sentry/internal_sdk.h"
+#include "sentry/level.h"
+
+#include <godot_cpp/core/object.hpp>
+#include <memory>
+
+using namespace godot;
+
+// Entry point for Sentry SDK users.
+// This singleton class exposes the public API of the Sentry SDK for the Godot Engine.
+class SentrySDK : public Object {
```

Comment:
@bitsandfoxes 
I went with this styling, because acronyms/initialisms are all capitalized in Godot with PascalCase, unlike C#/MSDN convention. For example:
- [ORMMaterial3D](https://docs.godotengine.org/en/stable/classes/class_ormmaterial3d.html)
- [JSON](https://docs.godotengine.org/en/stable/classes/class_json.html)
- [VisualShaderNodeScreenUVToSDF](https://docs.godotengine.org/en/stable/classes/class_visualshadernodescreenuvtosdf.html)
- [AudioStreamWAV](https://docs.godotengine.org/en/stable/classes/class_audiostreamwav.html)
- [SkeletonModification2DFABRIK](https://docs.godotengine.org/en/stable/classes/class_skeletonmodification2dfabrik.html)